source/frontend/src/hooks/useBcAssessmentLayer.ts

```diff
@@ -46,29 +59,29 @@ export const useBcAssessmentLayer = (
   const getLegalDescriptionsWrapper = useWfsLayer(
     url,
     {
-      name: names['LEGAL_DESCRIPTION'],
+      name: names[BC_ASSESSMENT_TYPES.LEGAL_DESCRIPTION],
       withCredentials: true,
     },
     { throwError: true },
   );
   const getAddressesWrapper = useWfsLayer(url, {
-    name: names['ADDRESS'],
+    name: names[BC_ASSESSMENT_TYPES.ADDRESSES],
     withCredentials: true,
   });
   const getValuesWrapper = useWfsLayer(url, {
-    name: names['VALUE'],
+    name: names[BC_ASSESSMENT_TYPES.VALUES],
     withCredentials: true,
   });
   const getChargesWrapper = useWfsLayer(url, {
-    name: names['CHARGE'],
+    name: names[BC_ASSESSMENT_TYPES.CHARGES],
```

Comment:
Also want to confirm that the changes from hardcoded singular 'ADDRESS' / 'VALUE' etc to the enum values (which are all plural) would not affect the layer query params?

Reply:
Correct, the query params are just for the folio id and roll number. These enum values just correspond to the keys in the response object.

Reply:
okay but I mean that the final text is different. It used to be 'CHARGE' and now it will be 'CHARGE(S)' - would that still work?

Reply:
I'm just comparing the old hardcoded strings with the string values of the new enum and they don't match